modules/gke-cluster-standard/variables.tf

```diff
@@ -168,6 +168,13 @@ variable "enable_addons" {
   nullable = false
 }
 
+variable "enable_cilium_clusterwide_network_policy" {
```

Comment:
Please add this as a field inside `enable_features`

Reply:
Fixed in 5cc9dc635c74bb82dd995a39f399cd705f73a948 :pray: